charts/super-agent-deployment/templates/rbac.yaml

```diff
@@ -0,0 +1,42 @@
+{{- if .Values.rbac.create -}}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    {{- include "newrelic.common.labels" . | nindent 4 }}
+  name: {{ include "newrelic.common.naming.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+rules:
+  - apiGroups:
+      - notification.toolkit.fluxcd.io
+      - source.toolkit.fluxcd.io
+      - helm.toolkit.fluxcd.io
+      - image.toolkit.fluxcd.io
+      - kustomize.toolkit.fluxcd.io
```

Comment:
I know it is early to speak about this, but we are not expecting to create notification, image or customize CR

Reply:
Fair enough/ Removed on [#51cefaa](https://github.com/newrelic/helm-charts/pull/1155/commits/51cefaa5326aac44f1e1754647f953d3aed05a87)